ingest/annotations.py

```diff
@@ -256,3 +270,22 @@ def validate_format(self):
                 self.validate_against_header_count(),
             ]
         )
+
+    def validate_numeric_annots(self):
+        """Check numeric annotations are not string dtype
+        """
+        valid = True
+        for annot_header in self.file.columns[0:]:
+            annot_name = annot_header[0]
+            annot_type = annot_header[1]
+            if (
+                annot_type == 'numeric'
+                and self.file[annot_name].dtypes[annot_type] == 'O'
```

Comment:
Are we sure this is going to work reproducibly?  I wasn't sure what this line was checking, so I looked at the `pandas` docs, and then I tried some local testing.  Instead of getting back `'0'` for a string column, I get back `dtype('0')`, which is a `numpy.dtype`.

Example:

```
>>> d = {'col1': [1, 2], 'col2': [3, 4], 'col3': ['foo', 'bar']}
>>> df = pd.DataFrame(data=d)
col1  col2 col3
0     1     3  foo
1     2     4  bar
>>> df.dtypes
col1     int64
col2     int64
col3    object
dtype: object
>>> df.dtypes['col3']
dtype('O')
```

Reply:
The type you're getting back is a capital O for "object". I've checked and the conditional works if I check for 'object' rather than 'O' so I've updated the function for clarity. The line is checking that the study owner has declared the metadata to be a numeric type but pandas dtype inference has decided it can only be an object type [pandas order of type checking in inference, int -> float (-> bool) -> object]